drivers/serial/uart_ite_it8xxx2.c

```diff
@@ -128,8 +151,41 @@ static void uart_it8xxx2_rx_refresh_timeout(struct k_work *work)
 static int uart_it8xxx2_init(const struct device *dev)
 {
 	const struct uart_it8xxx2_config *const config = dev->config;
+	struct uart_it8xxx2_data *data = dev->data;
 	int status;
 
+	if (data->baud_rate > IT8XXX2_BAUDRATE_115200) {
```

Comment:
This can only set the baud rate from the devicetree.  How will the baud rate get set if the user calls `uart_configure()` at runtime?

Reply:
You are right, the runtime aspect needs to be considered. It has been modified.